modules/dynamic-plugins/con-preinstalled-dynamic-plugins.adoc

```diff
@@ -2,9 +2,17 @@
 
 = Preinstalled dynamic plugins
 
-{product} is preinstalled with a selection of dynamic plugins. The dynamic plugins that require custom configuration are disabled by default.
+{product} is preinstalled with a selection of dynamic plugins. For a complete list of dynamic plugins that are preinstalled in this release of {product-short}, see xref:rhdh-supported-plugins[Preinstalled dynamic plugin descriptions and details].
 
-For a complete list of dynamic plugins that are preinstalled in this release of {product-short}, see the xref:rhdh-supported-plugins[Dynamic plugins support matrix].
+The following preinstalled dynamic plugins are enabled by default:
+
+* `@janus-idp/backstage-plugin-analytics-provider-segment`
+* `@janus-idp/backstage-scaffolder-backend-module-quay`
+* `@janus-idp/backstage-scaffolder-backend-module-regex`
+* `@backstage/plugin-techdocs-backend`
+* `@backstage/plugin-techdocs`
+
+The dynamic plugins that require custom configuration are disabled by default.
```

Comment:
I suppose we can remove this since it's not just the ones with config that will be disabled by default in 1.3.

```suggestion
```